perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/pages/steps/SummaryStep.java

```diff
@@ -862,6 +862,14 @@ public void onClick(ClickEvent event) {
 				modal.setWidth("750px");
 
 
+				Button buttonContAnyway = new Button(translation.continueAnyway());
+				buttonContAnyway.setType(ButtonType.PRIMARY);
+				buttonContAnyway.addClickHandler(new ClickHandler() {
+					@Override
+					public void onClick(ClickEvent event) {
+						redirectAfterTimeout(redirectTo);
```

Comment:
Since we have 7s timeout before the redirect happens, I suggest to hide the modal (in my case message and spinner was hidden below the modal).

Reply:
Modal should hide now.